pkg/kubelet/kuberuntime/security_context.go

```diff
@@ -41,9 +40,6 @@ func (m *kubeGenericRuntimeManager) determineEffectiveSecurityContext(pod *v1.Po
 		return nil, err
 	}
 
-	// set ApparmorProfile.
-	synthesized.ApparmorProfile = apparmor.GetProfileNameFromPodAnnotations(pod.Annotations, container.Name)
```

Comment:
AppArmor has not graduated yet so I think we still need support for the annotations.

cc @pjbgf

Reply:
Yep, my understanding was that AppArmor annotations would have a similar deprecation strategy as the Seccomp had, in which the annotations were supported for at leat 2 minors after the new field types were introduced. Please refer to the section Version Skew Strategy within the AppArmor graduation KEP: https://github.com/kubernetes/enhancements/pull/3298

Reply:
I see. so for now we should wait for the AppArmor to graduate, then we look into this PR, or I close it now and file another one when it's graduate.